pkg/release/build.go

```diff
@@ -86,6 +90,8 @@ func TargetTripleFromPlatform(platform Platform) string {
 	switch platform {
 	case PlatformLinux:
```

Comment:
Nit: can do `case PlatformLinux, PlatformLinuxFIPS:` instead of duplicating the arm.